### PR TITLE
add twitter share button

### PR DESCRIPTION
**Pull request by xvvvyz**
#1

### CODE REVIEW

File: app/assets/javascripts/components/QuotesBox.js.jsx [hunk 1]
```diff
@@ -108,6 +108,9 @@ class QuotesBox extends React.Component{
             <div className="btn btn-success grey darken-1 shadows fix-that-width" onClick={this.nextQuote}>Next</div>
             <div className="row">
               {favorite_icon}
+              <a href={`https://twitter.com/intent/tweet?text=${quote}`} target="blank">
+                <i className="fa fa-twitter fa-3x" onClick={this.shareToTwitter}></i>
```
Comment (goelrohan6) [inline app/assets/javascripts/components/QuotesBox.js.jsx]: why have you used onClick here after href above ?

Comment (xvvvyz) [reply app/assets/javascripts/components/QuotesBox.js.jsx]: oops

File: app/assets/javascripts/components/QuotesBox.js.jsx [hunk 2]
```diff
@@ -108,6 +108,9 @@ class QuotesBox extends React.Component{
             <div className="btn btn-success grey darken-1 shadows fix-that-width" onClick={this.nextQuote}>Next</div>
             <div className="row">
               {favorite_icon}
+              <a href={`https://twitter.com/intent/tweet?text=${quote}`} target="blank">
```
Comment (goelrohan6) [inline app/assets/javascripts/components/QuotesBox.js.jsx:111]: Why have you hardcoded the whole href ?

Comment (xvvvyz) [reply app/assets/javascripts/components/QuotesBox.js.jsx:111]: I used a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

**Pull request closed without merging**